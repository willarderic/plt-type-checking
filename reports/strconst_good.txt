TOKEN: STRCONST  LEXEME: "hello kitty"				TOKEN: STRCONST  LEXEME: "hello kitty"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
START -> EXPR							START -> EXPR

---- Completed parsing ----					---- Completed parsing ----

