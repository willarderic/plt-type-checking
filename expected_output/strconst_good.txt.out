TOKEN: STRCONST  LEXEME: "hello kitty"
CONST -> STRCONST
EXPR -> CONST
START -> EXPR

---- Completed parsing ----

